Makefile

```diff
@@ -0,0 +1,24 @@
+BINARY=argocd-vault-plugin
+VERSION=0.1
+OS_ARCH=darwin_amd64
+
+default: build
+
+format: 
+	go fmt github.com/IBM/argocd-vault-plugin/...
```

Comment:
I don't think we need this, unless we are stopping the build on code that is not in the correct format https://github.com/spf13/cobra/blob/master/Makefile#L16

Reply:
You're right, I put that in there thinking it was a linter not a formatter

Reply:
Do we want to fail on bad format?

Reply:
I dont think we need that now. Most IDEs have auto formatting anyways.

Reply:
Note for Josh: setup auto format in VS Code